cpp/src/bitmask/null_mask.cu

```diff
@@ -222,6 +271,40 @@ cudf::size_type count_unset_bits(bitmask_type const *bitmask, size_type start,
   return (num_bits - detail::count_set_bits(bitmask, start, stop, stream));
 }
 
+// Create a bitmask from a vector of column views
+void concatenate_masks(std::vector<column_view> const &views,
+    bitmask_type * dest_mask,
+    cudaStream_t stream) {
+  using CDViewPtr =
```

Comment:
A bit nicer way to do this:
```
using CDViewPtr = std::result_of_t<column_device_view::create(decltype(v), decltype(stream));
```

Reply:
That is not working out for me. I get this compilation error.
```
workspace/cudf/cpp/src/bitmask/null_mask.cu(283): error: function "cudf::column_device_view::create" is not a type name
```